### DISCUSSION TITLE
Updating from v0.18.4 to v0.21.1 - scripts added in custom blocks content not working

### DISCUSSION THREAD

**Discussion by GayathriGuru** [GrapesJS/grapesjs#5119] [Q&A]
I am trying to include a custom chart in my application using GrapesJS and when I use the same code as shown in the image below,it works perfectly fine and as expected but when I use the same code in the updated version it breaks.Suggest me on how to proceed further.

The code I am currently using :

editor.BlockManager.add('chartjs',
    {
      id:'chartJS',
      label:'ChartJS',
      category: 'Basic',
      content: `<div>
      <canvas id="canvas"></canvas>
    </div>
    <style>
    .container {
      width: 80%;
      margin: 15px auto;
    }
    
    h2 {
      text-align: center;
    }
    </style>
    <script> 
    var ctx = document.getElementById("canvas").getContext('2d');
var barChart = new Chart(ctx, {
  type: 'bar',
  data: {
    labels: ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"],
    datasets: [{
      label: 'data-1',
      data: [12, 19, 3, 17, 28, 24, 7],
      backgroundColor: "rgba(255,0,0,1)"
    }, {
      label: 'data-2',
      data: [30, 29, 5, 5, 20, 3, 10],
      backgroundColor: "rgba(0,0,255,1)"
    }]
  }
});
    </script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.4/Chart.min.js"></script>`,
      draggable: true,
      removable: true,
      attributes:
      {
        class:'fa fa-chart'
      },
    });
  }

The expected response in v0.18.4 :

![Screenshot from 2023-05-17 16-07-16](https://github.com/GrapesJS/grapesjs/assets/91883238/fb4c1ef7-6244-4a5a-a22c-9f2b230eb90f)

But the response I receive in v0.21.1 : 

![Screenshot from 2023-05-17 16-08-36](https://github.com/GrapesJS/grapesjs/assets/91883238/ef043154-c4fd-4282-86fd-8ed71441b7ce)

Update:

I have now narrowed down the issue to the script not getting triggered.So,I am just trying to figure how to load the script.I have tried to add it in to the component as mentioned in the documentation.Can you help me out here?

**Comment by artf** [comment]
Follow the documentation [here](https://grapesjs.com/docs/modules/Components-js.html) how to properly register components with JS
